### PR TITLE
update: AZ uses arcgis instead of tableau

### PR DESCRIPTION
**Pull request by cmcjacob**

### CODE REVIEW

**Comment by chunder** [comment]
Does the CSV contain any timestamps showing when the data was last updated? I'm trying to make sure our crawlers can detect when they are scraping stale data.

@lazd and I were talking about having scrapers throw an error if the data they find was last updated more than 1 day ago.

**Comment by cmcjacob** [comment]
Unfortunately this state is not adding update timestamps to [their dashboard](http://azdema.maps.arcgis.com/apps/opsdashboard/index.html#/331b8b9d7a764017a21815f6734e11ff) yet.  Only a handful are, currently.  Could stale data be detected from unchanging data in cases we don't have update timestamps?

File: src/shared/scrapers/USA/AZ/index.js [hunk 1]
```diff
@@ -18,35 +18,53 @@ const scraper = {
...
 
-      counties.push({
-        county,
-        cases
-      });
-    }
+      counties.push(transform.sumData(counties));
+      return counties;
+    },
+    '2020-3-20': async function() {
```
Comment (lazd) [inline src/shared/scrapers/USA/AZ/index.js]: did you mean `2020-3-30`?

File: src/shared/scrapers/USA/AZ/index.js [hunk 2]
```diff
@@ -18,35 +18,53 @@ const scraper = {
...
+          // unfortunately even arcgis isnt reporting any death data
+          county: county.NAME,
+          cases: parse.number(county.Number_Confirmed || 0)
+        });
+      }
 
-    counties.push(transform.sumData(counties));
-    return counties;
+      counties.push(transform.sumData(counties));
+      console.log(counties);
```
Comment (lazd) [inline src/shared/scrapers/USA/AZ/index.js]:
```suggestion
```

**Pull request merged**